recipes/pywinpty/bld.bat

```diff
@@ -0,0 +1,4 @@
+
+set DISTUTILS_USE_SDK=1
```

Comment:
`conda-build` already does [this]( https://github.com/conda/conda-build/blob/2.1.15/conda_build/windows.py#L102 ). So should be possible to drop this line and inline the install step in `meta.yaml` under [`script`]( https://conda.io/docs/building/meta-yaml.html#script ).